src/types.rs

```diff
@@ -17,9 +14,13 @@ use crate::ModuleSpecifier;
 use crate::SourceRange;
 use crate::SourceRangedForSpanned;
 use crate::SourceTextInfo;
+use deno_error::JsError;
+use std::borrow::Cow;
+use std::fmt;
```

Comment:
Why move this here? One day we'll get this formatting here: https://github.com/denoland/deno/pull/26646

Reply:
i didnt, cargo fmt did

Reply:
Lies.